introduction.md

```diff
@@ -1,8 +1 @@
 My name i carl nyden and i am quite new to github but i am quite quick to learning and will be quite usfull. my favorite path is the Html and the CSS branch
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?